src/Http/Headers/src/Microsoft.Net.Http.Headers.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <Reference Include="Microsoft.Extensions.Primitives" />
+    <Reference Include="Microsoft.AspNetCore.Http.Features" />
```

Comment:
> Did you do it this way because of the Http.Features netstandard2.0 target?

* `Http.Headers` is `net6.0` only
* `Http.Features` is `net6.0`, `netstd2.0` and `net461`

> Ideally Http.Features would depend on Http.Headers,

3 options there?

* Drop support for `netstd2.0` and `net461` in `Http.Features` 
* Add support for `netstd2.0` and `net461` in `Http.Headers` 
* Conditionally reference `Http.Headers` but only for the `net6.0`  version of `Http.Features`

Reply:
Option I'd go for is removing `netstd2.0` and `net461`...

Reply:
The SignalR client consumes the Features package so we can't drop TFMs from it.

> * Add support for `netstd2.0` and `net461` in `Http.Headers`

How bad is it? I assume it's using APIs not available on net461, but I can't guess how many. This isn't a library we've aggressively optimized.

> * Conditionally reference `Http.Headers` but only for the `net6.0`  version of `Http.Features`

So the IHeadersDictionary partial would be conditional to net6.0? Does that break anything upstream?

Reply:
> So the IHeadersDictionary partial would be conditional to net6.0? 

Well neither `netstd2.0` or `net461` support default interface methods so it would have to be.

Reply:
https://github.com/dotnet/aspnetcore/issues/31723 will address this by splitting up the Http.Features assembly.

Reply:
After #32043 please switch this around so Http.Features depends on Http.Headers and revert the type move.